### ISSUE TITLE
Add an installer

### ISSUE THREAD

**Issue by bNobo** [bNobo/needabreak#6]
Create an installer in order to permit end-user to easily install and update the application.

**Comment by bNobo** [comment]
It raises several questions : 

* Which technology is best suited : ClickOnce, WIX, Visual Studio Project template, ... The main con of ClickOnce is that it requires a digital certificate.
* Where to store the setup and binaries to make them available to everyone ?
* How to update the application once installed ?
* How to communicate about the availability of this free application ?

**Comment by bNobo** [comment]
Advanced Installer is definitively not an option: it is a commercial product, the free version is too limited for my needs. For instance the free version do not permit to automatically launch the application after installation.

**Comment by bNobo** [comment]
In order to store binaries or setup, Github recommand using Releases : https://help.github.com/en/github/managing-large-files/distributing-large-binaries

There also are several other solutions:
* https://pages.github.com/
* https://www.softpedia.com/user/contact.php
* https://www.filehorse.com/submit/
* https://www.snapfiles.com/feedback/

**Comment by bNobo** [comment]
I will stick using ClickOnce because it is the most convenient way to create a setup and easily allow auto-update of the application. I'll have to buy an Authenticode digital certificate one day in order to get rid of the UAC message.

I just published a v1.0.0 release with setup files attached in a zip. I also created a GitHub page to hold the update location used by ClickOnce. It should allow auto-update.

I encountered an interesting problem while testing : the setup once uploaded to git did not work anymore. It was because the ClickOnce manifest is hashed with CRLF line endings and git replace CRLF with LF during commit. The hash did not match the file anymore and the setup was considered corrupted. I have solved the problem with this command :

`git config --local core.autocrlf false`

It disables the automatic CRLF correction for the project.
